app/views/planning_application/consultations/show.html.erb

```diff
@@ -26,6 +26,24 @@
 
     <%= render "form", planning_application: @planning_application, consultation: @consultation %>
 
+    <% if flash[:sent_neighbour_letters].present? %>
```

Comment:
I'm wondering if this should be at the top of the page so it's more obvious?

Reply:
That's what the [GOV.UK Design System](https://design-system.service.gov.uk/components/notification-banner/#how-it-works) says, and it would match how we normally use flash messages; but the [design](https://www.figma.com/proto/l6yzC3qHvmgeYuSFzC6YWp/Consultations?type=design&node-id=239-15949&scaling=min-zoom&page-id=239%3A15793) says it should be just before the list.

Reply:
Ah I see. But that design doesn't have the map on it, so I think it'd end up being quite far down the page? Maybe worth asking a designer

Reply:
Ah, I see — because the design doesn't have the map, I've interpreted it as "right above the list" because that's the same as "right below the header details". I think if I put it right below the header but above the map it makes sense in the context of the design _and_ the added map.